src/app/components/formular-gui/formular-gui.component.html

```diff
@@ -0,0 +1,122 @@
+      <textarea [id]="formModel.id" [rows]="formModel.lines" [ngModel]="formModel.value" (ngModelChange)="onModelChange($event, formModel.id)"></textarea>
+    </div>
+
+    <div class="row" *ngSwitchCase="'Combobox'">
+      <label for="formModel.id">{{ formModel.value }}</label>
+      <select [id]="formModel.id" class="chosen-select" [ngModel]="formModel.value" (ngModelChange)="onModelChange($event, formModel.value)">
+        <option *ngFor="let option of formModel.options" [id]="option.id" [value]="option.value">{{ option.value }}</option>
+      </select>
+    </div>
+
+    <div class="row" *ngSwitchCase="'Separator'">
+      <hr>
+    </div>
+
+    <div class="row" *ngSwitchCase="'Hbox'">
+      <ul class="flex-container hbox">
+        <div *ngFor="let control of formModel.controls" [ngSwitch]="control.constructor.name">
+          <li class="flex-item" *ngSwitchCase="'Button'">
+            <button class="btn btn-primary" [id]="control.id" [type]="control.type" [ngModel]="control.value" (ngModelChange)="onModelChange($event, control.id)">{{ control.value }}</button>
+          </li>
+
+          <li class="flex-item" *ngSwitchCase="'Textfield'">
+            <label for="control.id">{{ control.value }}</label>
+            <input [id]="control.id" [type]="control.type" [ngModel]="control.value" (ngModelChange)="onModelChange($event, control.id)">
+          </li>
+
+          <li class="flex-item" *ngSwitchCase="'Label'">
+            <label [id]="control.id"></label>
+          </li>
+
+          <li class="flex-item" *ngSwitchCase="'Checkbox'">
+            <label for="control.id">{{ control.value }}</label>
+            <input type="checkbox" [id]="control.id" [ngModel]="control.value" (ngModelChange)="onModelChange($event, control.id)">
+          </li>
+
+          <li class="flex-item" *ngSwitchCase="'Textarea'">
+            <label for="control.id">{{ control.value }}</label>
+            <textarea [id]="control.id" [rows]="control.lines" [ngModel]="control.value" (ngModelChange)="onModelChange($event, control.id)"></textarea>
+          </li>
+
+          <li class="flex-item" *ngSwitchCase="'Combobox'">
+            <label for="control.id">{{ control.value }}</label>
+            <select [id]="control.id" class="chosen-select" [ngModel]="control.value" (ngModelChange)="onModelChange($event, control.value)">
+              <option *ngFor="let option of control.options" [id]="option.id" [value]="option.value">{{ option.value }}</option>
+            </select>
+          </li>
+
+          <li class="flex-item" *ngSwitchCase="'Separator'">
+            <hr>
+          </li>
+        </div>
+      </ul>
+    </div>
+
+    <div class="row" *ngSwitchCase="'Tabs'">
+      <tabset [pills]="true">
+        <div *ngFor="let tab of formModel.controls">
+          <tab [title]="tab.title">
+            <div *ngFor="let control of tab.controls" [ngSwitch]="control.constructor.name">
```

Comment:
Alle Controls wiederholen sich -> duplicate Code. Es gibt zwei Möglichkeiten das zu lösen:
* Den Inhalt der For-Schleife in eine eigene Komponente auslagern
* Ein Template für den Inhalt der For-Schleife anlegen, und dieses entsprechend einbinden